conbench/templates/index.html

```diff
@@ -54,28 +54,43 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
                      </td>
 
                      <td>
+                         <div class="table-entry">
                        {% if run.commit.author_avatar %}
-                         <image src="{{ run.commit.author_avatar }}" height="30" style="border-radius: 50%;">&nbsp;
+                         <img src="{{ run.commit.author_avatar }}" height="30" style="border-radius: 50%;">&nbsp;
                        {% endif %}
                        {{ run.commit.author_name }}
+                         </div>
                      </td>
 
-                     <td><a href="{{ run.commit.repository }}">{{ run.commit.display_repository }}</a></td>
+                    <td>
+                         <div class="table-entry">
+                             <a href="{{ run.commit.repository }}">{{ run.commit.display_repository }}</a>
+                         </div>
+                    </td>
 
-                     {% if run.commit.display_message %}
-                       <td><a href="{{ run.commit.url }}">{{ run.commit.sha[:7] }} {{ run.commit.display_message }}</a></td>
-                     {% else %}
-                       <td><a href="{{ run.commit.url }}">{{ run.commit.sha }}</a></td>
-                     {% endif %}
+                    <td>
+                         <div class="table-entry">{{ run.hardware.name }}</div>
```

Comment:
You may still need the ellipsis property here; I think hardware names could be long (see the [arrow conbench](https://conbench.ursa.dev/) for example)

Reply:
The "table-entry" class has the "fit-width" property, that should cover this use case. 

To validate: screenshots from CSS editors (picked a random long hardware name)

Using "table-entry" class
<img width="276" alt="Screenshot 2022-12-22 at 11 26 05 AM" src="https://user-images.githubusercontent.com/17350312/209067235-86cb4eaa-a0a9-4363-89f5-da5849e00b85.png">

Using "ellipsis-175" class
<img width="199" alt="Screenshot 2022-12-22 at 11 27 41 AM" src="https://user-images.githubusercontent.com/17350312/209067283-9efa9e54-8a18-4630-ac83-47a1fe698112.png">

Reply:
> I think hardware names could be long (see the [arrow conbench](https://conbench.ursa.dev/) for example)

Hm. May there be a misunderstanding here? I believe that Austin's point was that we do want this column to have a max width, and not have it consume the full width of very long hardware names.